bitbots_vision/src/bitbots_vision/vision_modules/lines.py

```diff
@@ -146,12 +146,34 @@ def get_line_mask(self):
             not_green_mask = np.ones_like(green_mask) - (np.floor_divide(green_mask, 255))
             # Get part under the field boundary as white mask
             field_boundary_mask = self._field_boundary_detector.get_mask()
+            # Shift mask up- or downwards depending on field boundary offset
+            shifted_field_boundary_mask = self._shift_field_boundary_mask(field_boundary_mask, self._field_boundary_offset)
             # Get not green points under field boundary
-            possible_line_locations = cv2.bitwise_and(not_green_mask, not_green_mask, mask=field_boundary_mask)
+            possible_line_locations = cv2.bitwise_and(not_green_mask, not_green_mask, mask=shifted_field_boundary_mask)
             # Get white points that are not above the field boundary or in the green field
             self._white_mask = self._white_detector.mask_bitwise(possible_line_locations)
         return self._white_mask
 
+    def _shift_field_boundary_mask(self, mask, offset):
+        shape = mask.shape
+
+        if offset == 0:
+            return mask
+        elif offset < 0:
+            # Shift mask downwards
+            offset = min(-offset, shape[0]-1)
+            frame = np.zeros(shape, dtype=np.uint8)
```

Comment:
Theoretically, we could shift the matrix in place and fill the missing stripe with zeros. This should increase the performance quite a lot.

Reply:
This needs to be evaluated...